ui/components/component-library/label/label.stories.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import {
+  DISPLAY,
+  FLEX_DIRECTION,
+  COLORS,
+  SIZES,
+  ALIGN_ITEMS,
+} from '../../../helpers/constants/design-system';
+
+import Box from '../../ui/box';
+import { Icon, ICON_NAMES } from '../icon';
+import { TextFieldBase } from '../text-field-base';
+
+import { Label } from './label';
+
+import README from './README.mdx';
+
+export default {
+  title: 'Components/ComponentLibrary/Label',
+  id: __filename,
+  component: Label,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    htmlFor: {
+      control: 'text',
+    },
+    required: {
+      control: 'boolean',
+    },
+    disabled: {
+      control: 'boolean',
+    },
+    children: {
+      control: 'text',
+    },
+    className: {
+      control: 'text',
+    },
+  },
+  args: {
+    children: 'Label',
+  },
+};
+
+const Template = (args) => <Label {...args} />;
+
+export const DefaultStory = Template.bind({});
+DefaultStory.storyName = 'Default';
+
+export const Children = (args) => (
+  <Box display={DISPLAY.FLEX} flexDirection={FLEX_DIRECTION.COLUMN} gap={2}>
+    <Label {...args}>Plain text</Label>
+    <Label {...args} display={DISPLAY.FLEX} alignItems={ALIGN_ITEMS.FLEX_START}>
+      Text and icon
+      <Icon
+        color={COLORS.ICON_ALTERNATIVE}
+        name={ICON_NAMES.INFO_FILLED}
+        size={SIZES.AUTO}
+      />
+    </Label>
```

Comment:
Should we add a `<Label><Input /></Label>` example?

Reply:
Added below